### PR TITLE
workon now creates new branch from default branch instead of master

### PR DESCRIPTION
**Pull request by tijlk**

### CODE REVIEW

File: src/doing/pr/create_pr.py [hunk 1]
```diff
@@ -77,8 +77,12 @@ def cmd_create_pr(
     )
     remote_branches = [x.rpartition("/")[2] for x in remote_branches if x.startswith("refs/heads")]
 
+    # Find the default branch from which to create a new branch and target the pull request to
+    cmd = f"az repos show --repository '{repo_name}' --org '{organization}' -p '{project}'"
+    default_branch = run_command(cmd)["defaultBranch"]
```
Comment (timvink) [inline src/doing/pr/create_pr.py:82]: Are we certain this always has a value? Let's add a fallback:

```python
run_command(cmd)["defaultBranch"]
```

with

```python
run_command(cmd).get("defaultBranch","refs/heads/master") 
```

**Review by timvink — CHANGES_REQUESTED** [review]
Can we also add a remark in `docs/reference/manual/create_pr.md`, saying it will by default detect the default branch and target the PR there.

File: src/doing/pr/create_pr.py (hunk 1 above)
Comment (tijlk) [reply src/doing/pr/create_pr.py]: fixed

**Comment by tijlk** [comment]
> Can we also add a remark in `docs/reference/manual/create_pr.md`, saying it will by default detect the default branch and target the PR there.

Added!

**Comment by timvink** [comment]
Thanks!

**Pull request merged**